### ISSUE TITLE
AutoLoad node with invalid path causes crash on project load.

### ISSUE THREAD

**Issue by MidZik** [godotengine/godot#21790]
**Godot version:** 3.1 alpha

**OS/device including version:** Windows 10

**Issue description:** If an autoload node you have defined has a path to a script that does not exist, Godot will crash when you try to open the project.

May be related to #21682.

**Steps to reproduce:** 

1. Create an autoload node to a certain script (old_path.gd).
2. Rename that script to something different (new_path.gd).
3. Close the project and try to re-open it. Godot is not able to open it, and crashes.

Note: The editor runs fine after step 2, although if you try to open the invalid path through the AutoLoad tab, it will also cause a crash.

Workaround: Open the godot.project file and remove the faulty autoload.

**Minimal reproduction project:** [AutoloadCrash.zip](https://github.com/godotengine/godot/files/2355904/AutoloadCrash.zip) (This project was created in 3.0.5, where it doesn't even output an error. When opened in 3.1-alpha, it crashes.)

**Comment by allkhor** [comment]
It's the same issue as #21790, can be closed as duplicate.

**Comment by vnen** [comment]
Closing as duplicate.

**Comment by lmerriam** [comment]
This exact bug still occurs for me in 3.1 stable. I don't believe this is a duplicate of #21682 which was closed.

**Comment by akien-mga** [comment]
Doesn't crash for me in 3.1-stable nor the master branch with the project in this issue.

If you have a similar crash in a different project, please open a new issue. Similar symptoms don't mean that it's the same bug (and this one was fixed in September 2018).
